packages/wonder-blocks-core/util/get-element-intersection.js

```diff
@@ -66,19 +66,29 @@ function getElementIntersectionAgainstParent(
     intersectingRect: ClientRect | DOMRect,
     boundsElement: Element,
 ): Intersection {
+    // We need to check that it matters if we're out of bounds. If the parent
+    // element isn't clipping or otherwise hiding things outside its bounds,
+    // then checking against bounds isn't going to be much use.
+    // So, let's get the style for the element and use the overflow values.
+    const style =
+        ((boundsElement: any).currentStyle: ?CSSStyleDeclaration) ||
+        window.getComputedStyle(boundsElement);
```

Comment:
TIL about `currentStyle`.  https://developer.mozilla.org/en-US/docs/Web/API/Element/currentStyle says it isn't part of any spec... I guess it doesn't matter since we also have a fallback.

Reply:
Yeah, from my understanding, it's quicker than `getComputedStyle` when it's available, so things often use it just in case.